tests/e2e/hellodapr_test.go

```diff
@@ -10,32 +10,26 @@ package e2e
 import (
 	"encoding/json"
 	"fmt"
-	"os"
 	"testing"
 
 	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
 	"github.com/dapr/dapr/tests/runner"
 	"github.com/stretchr/testify/require"
 )
 
-type testCommandRequest struct {
-	Message string `json:"message,omitempty"`
-}
-
-type appResponse struct {
-	Message   string `json:"message,omitempty"`
-	StartTime int    `json:"start_time,omitempty"`
-	EndTime   int    `json:"end_time,omitempty"`
-}
-
-var tr *runner.TestRunner
-
-func TestMain(m *testing.M) {
```

Comment:
I'm not sure about this change. Only a single TestMain is allowed per package test. So we can either create separate package for each test or use subtests with test runner setup in top level test.
I've also moved everything in a function scope to not collide common names like `testCommandRequest` with other tests.

Reply:
Thanks for your contribution :) I would create the separate package for this rather than this. 
I will update the doc and code properly today.

Reply:
Here is the pr to separate e2e package. - https://github.com/dapr/dapr/pull/848

Reply:
@youngbupark I've rebased my changes based on #848